google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/CloudApiManagementConfirmationDialog.java

```diff
@@ -42,6 +67,17 @@
   private JList<String> apisNotSelectedToEnableList;
   private JLabel enableConfirmationLabel;
   private JLabel wontEnableConfirmationLabel;
+  private JPanel apiEnablementPanel;
+  private JPanel serviceAccountPanel;
+  private JCheckBox newServiceAccountCheckbox;
+  private JPanel serviceAccountDetailsPanel;
+  private JScrollPane serviceAccountDetailsPane;
+  private JTable roleTable;
+  private JScrollPane rolePane;
+  private JPanel rolePanel;
+
+  private final Set<Role> roles;
+  private static final boolean UPDATE_SERVICE_ACCOUNT_DEFAULT = true;
 
   /**
    * Initializes the Cloud API management confirmation dialog.
```

Comment:
Could we also have `@param` javadoc descriptions for all parameters since we already have them for first two?

Reply:
done